package.json

```diff
@@ -48,14 +49,15 @@
     "grunt-newer": "^0.7.0",
     "grunt-sass": "^1.2.0",
     "highlight.js": "^8.0.0",
-    "karma": "^0.12.16",
-    "karma-chai-sinon": "^0.1.3",
+    "karma": "^1.3.0",
+    "karma-babel-preprocessor": "^6.0.1",
```

Comment:
Fine to bump karma and existing plugins, just no need to add karma-babel-preprocessor and related packages (babel-preset-es2015, and ?)